fix: Small edits for clarity (#747)
loft-sh/vcluster-docs: vcluster/configure/vcluster-yaml/sync/to-host/core/pods/hybrid-scheduling.mdx

## Use host schedulers

`sync.toHost.pods.hybridScheduling.hostSchedulers` specifies a list of schedulers available on the host cluster that virtual cluster pods can use.

The default Kubernetes scheduler on the host cluster is always implicitly included in the `sync.toHost.pods.hybridScheduling.hostSchedulers` list.  

> Any pod that uses the default scheduler—either by omitting `.spec.schedulerName` or by explicitly setting it to `default-scheduler`—is scheduled by the Kubernetes scheduler running on the host cluster.

Reviewer: People coming here will wonder how to use default scheduler, so we should put that idea first. Something like: 
```suggestion
A default scheduler can be used in the following ways: by omitting `.spec.schedulerName`, or by explicitly setting it to `default-scheduler`. This will then be scheduled by the default Kubernetes scheduler running on the host cluster.
```

Author: Thank you! Should these docs be in v.25 and .24?
